fr-portfolio.html

```diff
@@ -1,75 +1,79 @@
-	</section>	
-	<section id="about" class="about-alt">
-		<h2> About me </h2>
-			<div class="about-me">
-				<div>
-					<img src="https://via.placeholder.com/350x200">
-				</div>
-				<div>
-					<p>Hi this is where I should put some information about myself</p>
-				</div>
-			</div>		
-	</section>
-		
-	<section id="portfolio" class="projects">
-		<h2>Portfolio</h2>
-			<div class="portfolio-container">
-				<div>
-					<img src="https://via.placeholder.com/350x200">
-					<p>This is where my project information will go</p>
-				</div>
-				<div>
-					<img src="https://via.placeholder.com/350x200">
-					<p>This is where my project information will go </p>
-				</div>	
+ <main>	
+  <nav class="navbar">
+   <a href="#portfolio">Portfolio</a>
+   <a href="#about">About</a>
+   <a href="#contact">Contact</a>
+  </nav>
+	
+  <section class="welcome-banner" >
+   <h1>Freddie's Portfolio</h1>
+  </section>	
+	
+  <article id="about" class="about-alt">
+	<h2> About me </h2>
+		<div class="about-me-container">
+			<figure>
+				<img src="https://via.placeholder.com/350x200">
+			</figure>
+			<div>
+				<p>Hi this is where I should put some information about myself</p>
 			</div>
-			<div class="portfolio-container">	
-				<div>
-					<img src="https://via.placeholder.com/350x200">
-					<p>This is where my project information will go</p>
-				</div>	
-				<div>
-					<img src="https://via.placeholder.com/350x200">
-					<p>This is where my project information will go</p>
-				</div>
-			</div>		
-	</section>
-	<section id="contact" class="contact-me">
-		<h2>Contact</h2>
-			<div class="my-contacts">
-				<div>
-					<h3>Email</h3>
-					<i class="fas fa-at"></i>
-				</div>
-				<div>
-					<h3>LinkedIn</h3>
-					<i class="fab fa-linkedin"></i>
+		</div>		
+  </article>
+		
+  <section id="portfolio" class="projects">
+   <h2>Portfolio</h2>
+   <div class="portfolio-container"> <!-- Portfolio container left hand start -->
+     <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go</figcaption>
+	 </figure>
+	 <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go </figcaption>
+	 </figure>	
+	 </div>  <!-- Portfolio container left hand end -->
+	<div class="portfolio-container">	<!-- Portfolio container right hand start -->
+	 <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go</figcaption>
+	 </figure>	
+	 <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go</figcaption>
+	 </figure>
+	</div>	<!-- Portfolio container right hand end -->	
+  </section>
+	
+  <footer id="contact" class="contact-me">
+	<h2>Contact</h2>
+	<div class="my-contacts">
+		<div>
+			<h3>Email</h3>
+			<i class="fas fa-at"></i>
+		</div>
+		<div>
+			<h3>LinkedIn</h3>
+			<i class="fab fa-linkedin"></i>
```

Comment:
No opening tag for this `<section>`, might be why things are going a bit crazy when you view the page. Almost perfect indentation here, just need to move the `</footer>` back one, then everything forwards one to accommodate the opening `<section>` tag

Reply:
Fixed, I think